### PR TITLE
docs (Update): Add note about remove behavior

### PR DESCRIPTION
**Pull request by bucanero**
Fixes GRAPHQL-961

### CODE REVIEW

File: content/graphql/mutations/update.md [hunk 1]
```diff
@@ -28,7 +28,12 @@ type Post {
 }
 ```
 
-Dgraph automatically generates input and return type in the schema for the update mutation. Update mutation takes filter as an input to select specific objects. You can specify set and remove operation on fields belonging to the filtered objects. It returns the state of the objects after updation.
+Dgraph automatically generates input and return types in the schema for the update mutation. Update mutation takes `filter` as an input to select specific objects. You can specify `set` and `remove` operations on fields belonging to the filtered objects. It returns the state of the objects after updating.
```
Comment (aaroncarey) [inline content/graphql/mutations/update.md:31]: Suggested edit: "Update mutation takes" -> "Update mutations take"

**Review by JatinDevDG — APPROVED** [review]

File: content/graphql/mutations/update.md [hunk 2]
```diff
@@ -44,7 +49,9 @@ type UpdatePostPayload {
 }
 ```
 
-**Example**: Update set mutation using variables
+### Set
+
+**Example**: Update `set` mutation using variables
```
Comment (aaroncarey) [inline content/graphql/mutations/update.md:54]: Please add a colon after "variables"

**Review by aaroncarey — APPROVED** [review]
Great improvements to this topic! Approved with a few small comments.

File: content/graphql/mutations/update.md (hunk 1 above)
Comment (bucanero) [reply content/graphql/mutations/update.md]: Fixed

File: content/graphql/mutations/update.md (hunk 2 above)
Comment (bucanero) [reply content/graphql/mutations/update.md]: fixed

**Pull request merged**